### PR TITLE
Add `quantize`

### PR DESCRIPTION
**Pull request by rilut**
Hi, do you plan to add `quantize` to your lib?

I'm working on it but I still have a problem to make it work: https://github.com/vunb/node-fasttext/compare/master...rilut:feature/quantize?expand=1

It seems that my code failed to do something like:

    ./fasttext quantize -output ../test/data/cooking.model -input ../test/data/cooking.train.txt -qnorm -retrain -epoch 1 -cutoff 1000 -qout

Do you have any suggestion on that? Thanks

### CODE REVIEW

**Comment by rilut** [comment]
(This PR should be an Issue rather than a PR, but looks like I was writing on a wrong tab 😆 )

**Comment by vunb** [comment]
I have not yet plan to do. But you can create a PR to this repository 😄 
I should have definitely added some tests to this :/

Firstly, i will change travis/ci script to build the binary from source.
Thank you for contribution!

**Comment by rilut** [comment]
Btw I think I found the answer to my problem,

`static NAN_METHOD(train)` only calls `new Train` which calls `Train::Execute` which calls `wrapper_->train` which only calls

    fastText_.train(a);
    fastText_.saveModel();
    fastText_.saveVectors();
    return loadModel(a->output + ".bin");

So I need to call `fastText_.quantize` somewhere, right?

(Foolish me, I thought this lib only parses the args object into string args in `NAN_METHOD(train)` and `wrapper.cc`)

**Comment by vunb** [comment]
Yes, you can check their code here:

https://github.com/facebookresearch/fastText/blob/200be7582f69b6d91d26c92a131b4f8104694f56/src/main.cc#L82-L96

```cc
  FastText fasttext;
  // parseArgs checks if a->output is given.
  fasttext.loadModel(a->output + ".bin");
  fasttext.quantize(a);
  fasttext.saveModel();
```

**Comment by rilut** [comment]
Thanks man, got the `quantize` to work in this commit! I'll clean up my code in the next commit

**Comment by vunb** [comment]
@rilut Please increase the package version to `0.3.0` to see the result in travis 😄 
https://travis-ci.org/vunb/node-fasttext

**Comment by vunb** [comment]
@rilut I have already merge this PR https://github.com/vunb/node-fasttext/pull/9. Please consider to pull and merge this one from master.

**Comment by rilut** [comment]
So all tests are green, but maybe my code should be reorganized?

**Comment by vunb** [comment]
LGTM, you can reorganize your code later. 👍

File: src/wrapper.cc [hunk 1]
```diff
@@ -289,9 +296,27 @@ std::map<std::string, std::string> Wrapper::train(const std::vector<std::string>
...
+	}
+
+	std::cout << "Input  <<<<< " << a->input << std::endl;
+  	std::cout << "Output >>>>> " << a->output + ".bin" << std::endl;
+
+    // parseArgs checks if a->output is given.
+    fastText_.loadModel(a->output + ".bin");
+    fastText_.quantize(a);
+    fastText_.saveModel();
+    exit(0);
```
[the remark this answers is not in the document]
Comment (rilut) [reply src/wrapper.cc]: yeah I already do it on my latest commit, `return loadModel(a->output + ".ftz")`

[the remark this answers is not in the document]
Comment (vunb) [reply src/wrapper.cc]: I have seen!

**Pull request merged**